### ISSUE TITLE
media shortcuts stop working after suspending PC

### ISSUE THREAD

**Issue by lbausch** [phanan/koel-app#3]
I am running the latest koel-app (730c18d) on Debian sid. After suspending and starting the PC again, Koel no longer accepts media shortcuts. Specifically "pause", "next song" and "previous song" are no longer working. Other shortcuts like "volume up/down" are still functional, but these are handled by Debian itself.
Is this a possible bug in Koel or something Debian related?

**Comment by phanan** [comment]
Hmm. I'm not sure about your question honestly. Koel simply uses the [global shortcut registrar](https://github.com/atom/electron/blob/master/docs/api/global-shortcut.md) provided by Electron. I guess this has something to do with Electron/Debian instead of Koel.

**Comment by lbausch** [comment]
So this does not happen with OS X Yosemite?

**Comment by phanan** [comment]
No, it doesn't, at least for me. I can't recall the last time I turned off my MBP 😃

**Comment by lbausch** [comment]
ok, nevermind - thank you anyway :smile:

**Comment by phanan** [comment]
Actually I'd prefer leaving this issue open, maybe other users have a better idea/understanding than mine. Thanks for reporting this.
